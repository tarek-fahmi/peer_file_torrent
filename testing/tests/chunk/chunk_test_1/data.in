Confirm Chunks Validity
./testing/bin/pkg_main ./testing/resources/valid_1.bpkg -chunk_check
./testing/bin/pkg_main ./testing/resources/valid_2.bpkg -chunk_check