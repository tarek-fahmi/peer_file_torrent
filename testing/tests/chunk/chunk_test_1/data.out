e6f57f3830a5463a816a9ef343c2dd9ff4d53282459d942c2326e5d18af2d594
65a1a193f150de4166e3132f1edb77e07b15de5e368e9ec3dd9f45fe2abb07aa
e8f23fa8a41bf4d3fe337eff0b6e6bd54bf7277e9010bb95ecd4199ffcd6e97b
702b2a68ddea5d29cb3fe0a1a80189ca50eab29f4427ebb0002c5d8973c57f92
43996cc575d4aad804ea92ba28f211d157b545c922d689428beb37bab792e1df
a5de68ace533bc8e2d9ccc79b693d76b7349875b208d125bd13580b51beeb004
062f8f673892a6671d17cc8a12004b5caa380b9e6d2ea4d6bbc7e99b24b83e0d
67cbe6c40d72b42984675065a2fe0c27e522b4939be225fcbb9da84de6ed3d0a
36a7a745245fd57b4148c83cf67b211f2ac8ff08d702a0f87f0dabfdb140ae30
22482576645ee177beb7d43fe542b5d65f243afcd6a2e381934e1f13d00dc3ff
2dfad8fbfd7e3e49db275551b61ffbd3692d7760457d8dbf9984fb9536d82787
4c990a0ae24c09b5e253f0a04e9934557e116c197410bbbb4d2a690462007d07
e039f62113b787a152a7bdf90cc1c8fbf8140a063ba1f66c5e1674af430dbd1b
339d5a1e36954da5e4ec18773e19de93b7890139851b9de973a3af32c6c4bccb
0fd41e9a843b5a8d1f43068b1c2b203a96f85e3d66134505f82cae313328ab15
2ee88f55c7effd971f70205d3da2a082c17185b4d0c1e0c0ad3e3455132ec160
c5723651427a02aaba15432a53db56256a360c4882f669922c2a10dec5511147
02c8fae966d53b904af3216e11d55a34c99fe8659793714c4063eae673e1cfcc
e0733ea2325134bb62d8164aa64362aaa143949a575f96fb32d484364eed6230
7ec0d5d2158c1a0068b8b383f33ecac2104664cf10fb40e0cbcd8f1d752edf84
af9c2359ec229e490632272277c57b98a87859a478f3e3894ffc3f9a8c340a0a
fcbec7e8ca9e4988909f18349ab94531e6bef4621acc60f2f8736f91035d8a93
10f482aaff9de42d0785796e81a87285c2329e0944f8e7e233d1b707e745f738
d2077032fa9329fa241df4e63485a1984a7a1d875bfab9af40c1d1fbde926320
6f0444182ee2c64337a32a5ceff85435c7436902f7e5510c5cb020803488515b
a00093e3452b22efa4536bf553ffbaa21421c0a0de98199565a03f41fc72f755
4eafb15fc106679172bcaeaafb024fae2dc7a4a2996dc3b381b176758fa454fb
0b02e5329b6c39e7417ca4ebafc8480291b5a56ba21eb15d684e98ea5ec99f8d
2ff38393cf12d39d83c813cbff072d21b4deb50626a5ea8f4aade5f9d0c57128
9169a60fefa9a381912fb104988560ac4ca9bd1b936c625ec00bb4f4f52db59f
573b76091145cf9343d894f1755b99dd8d5c697a8a05e4d6f544172c258c5c4b
0613126d9cb85f0755a40a17a0d8c46aa996c02049ac9497e094afefd096661d